Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.py

```diff
@@ -0,0 +1,301 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from AWSApiModule import *
+
+from datetime import datetime, date
+import urllib3.util
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+param = demisto.params()
+
+SERVICE = 'identitystore'
+IDENTITYSTOREID = param.get('IdentityStoreId')
+
+
+class DatetimeEncoder(json.JSONEncoder):
+    # pylint: disable=method-hidden
+    def default(self, obj):
+        if isinstance(obj, datetime):
+            return obj.strftime('%Y-%m-%dT%H:%M:%S')
+        elif isinstance(obj, date):
+            return obj.strftime('%Y-%m-%d')
+        # Let the base class default method raise the TypeError
+        return json.JSONEncoder.default(self, obj)
+
+
+def create_user(args, client):  # pragma: no cover
+    username = demisto.getArg('userName')
+    familyName = demisto.getArg('familyName')
+    givenName = demisto.getArg('givenName')
+    userEmail = demisto.getArg('userEmailAddress')
+    userDisplayName = demisto.getArg('displayName')
+
+    response = client.create_user(
+        IdentityStoreId=f'{IDENTITYSTOREID}',
+        UserName=f'{username}',
+        Name={
+            'FamilyName': f'{familyName}',
+            'GivenName': f'{givenName}'
+        },
+        Emails=[
+            {
+                'Value': f'{userEmail}',
+                'Type': 'work',
+                'Primary': True
+            },
+        ],
+        DisplayName=f'{userDisplayName}'
+    )
+    ec = {'AWS.IAMIdentityCenter.Users': response}
+    human_readable = tableToMarkdown('AWS IAM Identity Center Users', response)
+    return_outputs(human_readable, ec)
+
+
+def get_user(args, client):  # pragma: no cover
+    data = []
+    userName = demisto.getArg('userName')
+    response = client.list_users(
+        IdentityStoreId=f'{IDENTITYSTOREID}',
+        Filters=[
+            {
+                'AttributePath': 'UserName',
+                'AttributeValue': f'{userName}'
+            },
+        ]
+    )
+    rep = json.dumps(response)
+    repJSON = json.loads(rep)
+    datas = repJSON.get('Users', [])
+    for da in datas:
+        for user in response['Users']:
+            user_details = {
+                'UserName': user['UserName'],
+                'UserId': user['UserId'],
+                'Email': user['Emails'][0]['Value'],
+                'DisplayName': user['DisplayName']
+            }
+            userID = user['UserId']
```

Comment:
Is there a reason you only take the last **UserId**, since multiple users can be returned?

Reply:
Actually, there will be always one user returned not multiple. I'm fetching list of users from AWS then looping on that list to check if the queried user is available or not. So, always one userId will be returned so i can use it in listing group assigned to that user later on.